.github/workflows/reusable-tox.yml

```
@@ -400,6 +400,9 @@ jobs:
         !cancelled()
         && steps.tox-run.outputs.test-result-files != ''
       uses: codecov/test-results-action@44ecb3a270cd942bdf0fa8f2ce14cb32493e810a  # v1.0.3
+      # There is a bug in action which provokes it to fail during upload
+      # Related issue: https://github.com/codecov/codecov-action/issues/1794
+      continue-on-error: true
```

IMHO it may make sense to add `FIXME` remark and current date to the comment to keep track of this quirk which we probably want to revert later 🤔 
```suggestion
      # There is a bug in action which provokes it to fail during upload
      # Related issue: https://github.com/codecov/codecov-action/issues/1794
      # FIXME: set to `false' once linked issue is resolved (20250321)
      continue-on-error: true
```

---

And browse git log to get when the comment first appeared if it's updated multiple times... I prefer a more human-friendly inline comment.
Up to you though.